di_website/templates/datasection/data_section_page.html

```diff
@@ -0,0 +1,157 @@
+                  {% image random_quote.value.external_member_photo max-400x250 as profile_img %}
+                  <img src="{{ profile_img.url }}" alt="alt tag">
+                {% else %}
+                  <img src="img/fallback-image.jpg" alt="alt tag">
+                {% endif %}
+              </aside>
+            </div>
+          {% endif %}
+        {% endif %}
+      </div>
+      <div class="l-2up__col">
+        {# TODO:: Load correct dataset number when dataset branch is completed #}
+        <p class="m-text-major"><span>583</span> datasets</p>
+        <p class="para para--typeset">
+          {{ page.dataset_subtitle }}
+        </p>
+        {% if page.hero_link %}
+          <a href="{{ page.hero_link }}" class="button">View datasets</a>
+        {% endif %}
+      </div>
+    </div>
+  </div>
+</section>
+
+{% for block in page.data_support %}
+
+  <section class="section">
+    <div class="row row--narrow">
+      <div class="banner">
+        <div class="l-banner">
+          <div class="l-banner__aside">
+            <div class="banner__media">
+              {% if block.value.photo %}
+                {% image block.value.photo max-400x250 as photo %}
+                <img src="{{ photo.url }}" alt="alt tag">
+                <span class="banner__credit">
+                  {% if block.value.photographer and block.value.photo_credit %}
+                    Image by <a href="{{ block.value.photo_credit }}">{{ block.value.photographer }}</a>
+                  {% endif %}
+                </span>
+              {% else %}
+                <img src="img/fallback-image.jpg" alt="alt tag">
+              {% endif %}
+            </div>
+          </div>
+          <div class="l-banner__main">
+            <h3 class="banner__heading">{{ block.value.title }}</h3>
+            <p class="banner__excerpt">
+              {{ block.value.description | default:"" }}
+            </p>
+            <ul class="banner__list">
+              {% if block.value.telephone %}
+                <li>Telephone: {{ block.value.telephone }}</li>
+              {% endif %}
+
+              {% if block.value.email %}
+                <li>Email: <a href="mailto:">{{ block.value.email }}</a></li>
+              {% endif %}
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+{% endfor %}
+
+{% if related_pages %}
+<section class="section">
+  <div class="row row--narrow">
+    <h2 class="section__heading">More about</h2>
+    <div class="l-4up">
+
+      {% for page_item in related_pages %}
+      <a href="#" class="card">
+        {% if page_item.specific.hero_image %}
+          {% image page_item.specific.hero_image max-400x250 as photo %}
+          <div class="card__media" style="background-image: url({{ photo.url }});">
+          </div>
+        {% else %}
+          <div class="card__media" style="background-image: url('img/fallback-image.jpg');">
```

Comment:
Please use `load static` at the top and reference this fallback image as a static file. I'm getting a 404 when it wants to load.

Reply:
Now using static tag and profile placeholder image as default when no image is provided..